src/CrossEncoderSearch.py

```diff
@@ -0,0 +1,25 @@
+import os
+import re
+from sentence_transformers import CrossEncoder
+from sentence_transformers import SentenceTransformer
```

Comment:
@genkai10 do we need the sentence transformer?

Reply:
nope we don't , will remove it .